x-pack/plugins/uptime/public/components/monitor/synthetics/waterfall/components/use_bar_charts.ts

```diff
@@ -17,17 +17,16 @@ export const useBarCharts = ({ data = [] }: UseBarHookProps) => {
 
   useEffect(() => {
     if (data.length > 0) {
-      let chartIndex = 1;
+      let chartIndex = 0;
 
-      const firstCanvasItems = data.filter((item) => item.x <= CANVAS_MAX_ITEMS);
```

Comment:
0 index was accounted for in subsequent canvas charts, but not in the canvas data array. Another solution could be `item.x <= CANVAS_MAX_ITEMS - 1`, but I decided to have all the logic within the `data.forEach` block, as I noticed all the items within the the first canvas data array was repeated twice.